### ISSUE TITLE
no example code for the sortable-table-rows example?

### ISSUE THREAD

**Issue by phryneas** [clauderic/react-sortable-hoc#312]
Hi, I'm looking through the examples but can't find the code associated with this example: http://clauderic.github.io/react-sortable-hoc/#/react-virtualized/sortable-table-rows
Can you please point me in the right direction?

**Comment by phryneas** [comment]
Okay, I finally found a working example in their codebase:
https://github.com/bvaughn/react-virtualized/blob/master/docs/Table.md#rowrenderer

Could you please add a source to that example nonetheless so the next person has a better time finding this? :)

**Comment by theednaffattack** [comment]
Yeah especially since the react-virtualized Table example gives an error when you run it (I get sortBy undefined. Probably because context is undefined, probably because I can't figure out how to add items). So there's no way to start from their table and learn to integrate sortable.  I'm trying to figure out how to get this working but having very little luck.  I'm trying to expand outward from react-virtualized's minimal example linked above but that's a little TOO minimal. 

Please add row sorting code to examples.

**Comment by a-cabinti** [comment]
Yes, Please give us example code for sortable rows instead of cols

**Comment by chhornponleu** [comment]
please find example here
https://github.com/clauderic/react-sortable-hoc/blob/master/examples/virtual-table-columns.js

**Comment by phryneas** [comment]
@chhornponleu : That's for columns, not for rows.

**Comment by bonavida** [comment]
@clauderic Can you please add some example for sortable table rows? Like the one you have on the demos page.

**Comment by kedarv** [comment]
Did anyone find a source example of the row reordering?

**Comment by yashcr07** [comment]
For anyone looking for a sortable-table-row example, I have made [this](https://codesandbox.io/s/oj4wnkjjkq).

I still couldn't figure out the auto scrolling part, so if anyone knows how to achieve it please fork this example and implement it. Thanks
